### PR TITLE
fix: Updated broken links; Fixed typos

### PR DESCRIPTION
**Pull request by alexandrtovmach**

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
@DmitrijDN @EdikDolynskyi @k-r-s-n-v 
Hey guys 👋 
Found few issues 😉

File: src/pages/lectures/express-yourself-with-nodejs/index.mdx [hunk 1]
```diff
@@ -187,12 +187,12 @@ import audio from "../../../assets/audio/in_the_end.mp3";
     Рассмотрим на примере как же работает Web:
     <ol>
       <li><em>пользователь открыл браузер</em> / <strong>запустил клиент</strong></li>
-      <li><em>пользователь открыл <a href="https://alexandrtovmach.com/images/preloader.svg">ссылку</a></em> / <strong>по HTTP отправил GET запрос на сервер</strong></li>
+      <li><em>пользователь открыл <a href="https://alexandrtovmach.com">ссылку</a></em> / <strong>по HTTP отправил GET запрос на сервер</strong></li>
```
Comment (EdikDolynskyi) [inline src/pages/lectures/express-yourself-with-nodejs/index.mdx:190]: Change link to this
https://binary-studio-academy.github.io/stage-2/static/ryu-845240705695b54d820b710eac8d0d5d.png

Comment (alexandrtovmach) [reply src/pages/lectures/express-yourself-with-nodejs/index.mdx:190]: are you sure that this link to changing every gatsby build?

**Review by EdikDolynskyi — APPROVED** [review]

**Comment by EdikDolynskyi** [comment]
@alexandrtovmach Thanks a lot!

**Comment by alexandrtovmach** [comment]
@EdikDolynskyi if you want:
https://github.com/binary-studio-academy/stage-2/pull/25

**Pull request merged**